Dockerfile

```diff
@@ -9,9 +9,7 @@ ARG CONTAINER_USERID
 # Configure our user
 RUN usermod -u $CONTAINER_USERID frontend
 
-ADD src/api/Gemfile /obs/src/api/Gemfile
-ADD src/api/Gemfile.lock /obs/src/api/Gemfile.lock
-RUN chown -R frontend /obs/src/api
+ADD src/api/Gemfile* /obs/src/api/
```

Comment:
Nitpick. I would add here the `chown`, and leave it as:

`ADD --chown=frontend src/api/Gemfile* /obs/src/api/`